src/main/psxsdk/libapi/counter.c

```diff
@@ -1,11 +1,12 @@
 #include "common.h"
+#include "psxsdk/libapi.h"
 
 typedef struct {
     u16 rootCounter;
-    s16 unk2;
+    u16 : 16;
```

Comment:
Could you explain this and the other change to this struct?

Reply:
I didn't see any references to `unk2` but understand the field alignment is important. The size of a struct field can be overridden with a `:` for bit packing but it also allows anonymous struct members that take up space but can't be addressed directly in code. It's a way to preserve the layout without accidentally using `unk2` for something if it is actually intended to remain unused.

Reply:
Seems OK to me, this file is conflicted now BTW